common/include/pcl/PointIndices.h

```diff
@@ -6,21 +6,22 @@
 
 // Include the correct Header path here
 #include <pcl/PCLHeader.h>
+#include <pcl/types.h>
 
 namespace pcl
 {
   struct PointIndices
   {
+    using Indices = std::vector<index_t>;
```

Comment:
I'm wondering if semantically it makes more sense to call this simply `Type`. The enclosing type is already named `*Indices`.